front/src/modules/ui/input/url/components/URLTextInputDisplay.tsx

```diff
@@ -18,18 +18,33 @@ type OwnProps = {
 };
 
 export function InplaceInputURLDisplayMode({ value }: OwnProps) {
+  let type = 'url';
   function handleClick(event: MouseEvent<HTMLElement>) {
     event.stopPropagation();
   }
-
   const absoluteUrl = value
     ? value.startsWith('http')
       ? value
       : 'https://' + value
     : '';
+  if (
+    /^(http|https):\/\/(?:www\.)?linkedin.com(\w+:{0,1}\w*@)?(\S+)(:([0-9])+)?(\/|\/([\w#!:.?+=&%@!\-/]))?$/.test(
```

Comment:
let's extract these linkedin/x checks to standalone functions to ease reading and re-usability

Reply:
Let's actually move these check to a SocialLink component that would encapsulate RoundedLink